slides/k8s/operators.md

```diff
@@ -386,4 +386,4 @@ We should see at least one index being created in cerebro.
 
 - What if we want different images or parameters for the different nodes?
 
-*Operators can be very powerful, iff we know exactly the scenarios that they can handle.*
```

Comment:
That one is actually not a typo; I used the short-hand notation for "if and only if". But you're not the first person who thinks that it's a typo, so I will rephrase that so that it's more clear. Thank you for pointing it out! <3